test/pageobjects/BasePage.js

```diff
@@ -9,12 +9,22 @@ class BasePage {
   async title() { return this.$('.onfido-sdk-ui-PageTitle-titleSpan')}
   async subtitle() { return this.$('.onfido-sdk-ui-PageTitle-subTitle')}
   async backArrow() { return this.$('.onfido-sdk-ui-NavigationBar-iconBack')}
+  async hideLogo() { return this.$('.onfido-sdk-ui-Theme-noLogo')}
 
   copy(lang) { return locale(lang) }
 
   async clickBackArrow() {
     this.backArrow().click()
   }
+
+  async checkLogoIsHidden() {
+    try {
+      return this.hideLogo().isDisplayed()
+    } catch (e) {
+      console.log("Logo is present:", e)
+      return false
+    }
+  }
```

Comment:
This test sounds weird reading it out. Maybe naming it `this.noLogo` would be better & also more consistent.

You could also simplify this check by using chai's `assert`, e.g. 
```
assert.isTrue(this.hideLogo().isDisplayed(), 'Test Failed: No logo should be displayed')
``` 
Have a look at the face or crossDevice test specs if you want more usage examples. :)

Reply:
Thanks, fixed.